### ISSUE TITLE
Vivladi browser and page change caching issues

### ISSUE THREAD

**Issue by Impulseman45** [Nexus-Mods/web-issues#261]
Bug/error being reported:When going from one Nexus game site to the next, the new page starts to render and is visible, but it then reverts to the previous page. the only way to move to the next page is to close Vivaldi and then restart it. then the pages loads as it should. But as soon as I go to the next page, its the same issue again.
Steps to reproduce: Can be reproduced in my browser each and every time i access the new Nexus sites
Expected result: I should be able to move from one Nexus game mod site the other and not get thrown back to the page i just tried to leave.
Actual result: Unknown

**Comment by verenion** [comment]
Can you provide a link to the page you are on, and point out the link you are clicking?

I've tried replicating this, but I cannot.

**Comment by Impulseman45** [comment]
When I use the Nexus I view mods in list format descending order all time mods. It was a feature I always used on the classic site as I abhorred the front page and seeing some of the mods there, personal taste for sure. But when I go from one page, say Skyrim SE’s list of mods, and transition to Skyrim Classic or Fallout 4, the page shows the next page loading and then reverts to the previous page, but the url above shows it being parked at the new page, but it is still showing the previous page. I could more show it in a short video clip. If you would like me to do that and capture it happening I can do that. It could just be things at my end, but its only in the new site format and only in Vivaldi. I use Vivaldi because it has a dark mode that is much easier on my eyes over time versus looking at pale grey frames and the time.

Thanks for getting back to me, let me know if you want to capture the footage.

Impulseman45

**Comment by verenion** [comment]
Can you still replicate this issue?

**Comment by Impulseman45** [comment]
Yes, I can still repeat the issue, even with two updates to the Vivaldi browser. It’s got to be some kind of caching issue. But I have found a work around,
but not sure if it will still work once the site permanently switches to the new format.

As mentioned before, it was an issue with listing all new mods in descending order, and also having the new site or Vivaldi remember to take me to all
Files page and not to the most  Endorsed. The site/Vivaldi won’t remember to keep my choice of by date setting and always defaults to most endorsed.
In the end I took the url that was my choice to view the site, https://www.nexusmods.com/skyrimspecialedition/mods/latestmods/? And replaced the
www.nexusmods with rd.nexusmods, https://rd.nexusmods.com/skyrimspecialedition/mods/latestmods/?. This works for me, I get the mods in
descending order from newest to oldest, I have no caching issues and I can go from each of the game mods pages and view them as I did in the old
site format. I choose to use this format instead of the new formats that push front page content in my face, because to be quite frank, I am tired of seeing
Huge boob mods on the front page all the time. I don’t want to turn adult content off, I just choose not to use the front page and have this in my face
All the time. I have no issue with mods dealing with nudity, but this particular style of mod has become a plague on the site and I have to do things in a
Way that I don’t personally have to see them on every visit. Hence my preferences to view mods in a tiled or Chiclet format. If the images interest me
I click the mod, if it’s one of the for-mentioned big boob mods I click the author of the mod and block them, no matter what mods they have made. If they
are going to make mods like this I never want to see anything they make. Not how Nexus wants to view mods as a whole, but I personally feel the site
is getting a fairly negative reputation over this, but DarkOne sees no need to segregate these things.
Anyway, in closing, I have a fix I am using, but if the new site when it goes live removed the structure in which I view mods, well I guess I will be cutting back
My use of the site and use other sites that point to mods on the Nexus as a means of viewing new mods. I am sure I will miss some good mods, but I simply
Don’t want to view mods the way things are going, I don’t like using the front page, I don’t like the way some front page content follows along in certain views
of the site, hence me using the patch work url I am using. I am hoping it still will continue to work, but I fear it won’t as the new site goes live permanently.
Donald Edwards
Aka Impulseman45
